fabric-data-generation-api-v1/src/client/java/net/fabricmc/fabric/mixin/datagen/client/MinecraftClientMixin.java

```diff
@@ -27,7 +27,7 @@
 
 @Mixin(MinecraftClient.class)
 public class MinecraftClientMixin {
-	@Inject(method = "<init>", at = @At(value = "INVOKE", target = "Lcom/mojang/blaze3d/systems/RenderSystem;getBackendDescription()Ljava/lang/String;"))
+	@Inject(method = "<init>", at = @At(value = "INVOKE", target = "Lcom/mojang/blaze3d/systems/RenderSystem;getBackendDescription()Ljava/lang/String;", remap = false))
```

Comment:
I thought that Blaze 3D was obfuscated?

Reply:
Parts of it are not obfuscated for whatever reason, this includes RenderSystem iirc

Reply:
Yeah, this is not obfuscated, just double checked. 👍